plugins/extractors/metabase/metabase.go

```diff
@@ -103,33 +93,25 @@ func (e *Extractor) Extract(ctx context.Context, emit plugins.Emit) (err error)
 		// "ordered_cards" field
 		dashboard, err := e.buildDashboard(d)
 		if err != nil {
-			return errors.Wrap(err, "failed to fetch dashboard data")
+			e.logger.Error("failed to build dashboard with", "dashboard_id", d.ID, "err", err.Error())
+		} else {
```

Comment:
Any reason why we use `else` here?

Why not

```suggestion
		dashboard, err := e.buildDashboard(d)
		if err != nil {
			e.logger.Error("failed to build dashboard with", "dashboard_id", d.ID, "err", err.Error())
			return
		}
		emit(models.NewRecord(dashboard))
```

Reply:
oh no major reason really, was just trying to avoid using `continue`.
let's use your suggestion, but instead of return we'll use `continue`.

Reply:
I see, either way is fine though, just curious.

Reply:
> I see, either way is fine though

I like yours better 👌 

@mabdh updated it here
https://github.com/odpf/meteor/pull/265/commits/eacca61190efa465dba5f3a2de77c0bce4721671